### ISSUE TITLE
Need of checkpoints : BT and VicReg

### ISSUE THREAD

**Issue by KameniAlexNea** [DonkeyShot21/cassle#5]
Hi all,

Is there any link where I can access to the checkpoint of model trained using Barlowtwins and VicREG ?

I would like to evaluate this approach using different models and need the trained last checkpoint of these models.

Thanks.

**Comment by vturrisi** [comment]
Hey. I'm a bit busy this week, but I'll try to share the checkpoints as soon as possible.

**Comment by KameniAlexNea** [comment]
Hi, can you please share the configuration used to obtain results in `assets\results` file ?

I have tried to change some confs without getting it. Updating learning rate as mentioned in the paper.

Are you using online evaluation when training ? At the end of CSSL, what percentage of data are you using to train your linear model when trained on CIFAR-100 ?

**Comment by DonkeyShot21** [comment]
Hi, you can find the script in `bash_files/continual/{dataset}/{method}.sh`. They should contain the correct arguments. 

On CIFAR100 we report the online linear eval accuracy since offline eval does not improve. Except for table 9 (semi-supervised), we use 100% of the data to train the linear classifier.

**Comment by KameniAlexNea** [comment]
Thank you for your feedback. 

II apologize for the omission in the information provided. I think that this argument could be added in the files on CIFAR-100.
